lib/jxl/encode.cc

```diff
@@ -1425,12 +1426,15 @@ JxlEncoderStatus JxlEncoderAddJPEGFrame(
     }
   }
 
-  if (!frame_settings->enc->basic_info_set) {
+  if (!frame_settings->enc->basic_info_set || !io.blobs.exif.empty()) {
```

Comment:
I don't think we should overwrite basic info just because we have exif. In that case we should expect the orientation to be already set correctly.

Reply:
only overwriting orientation now